### PR TITLE
Remove version number from jar files

### PR DESCRIPTION
**Pull request by hernanponcedeleon**

### CODE REVIEW

**Comment by ThomasHaas** [comment]
Keep in mind that the CNA repo might also need an update to avoid the version number (if you want to keep that repo alive).

**Comment by hernanponcedeleon** [comment]
The CNA scrips checkout a concrete commit of dartagnan, so this should not cause any problem

File: dartagnan/pom.xml [hunk 1]
```diff
@@ -653,6 +653,7 @@
   </dependencies>
 
   <build>
+    <finalName>${project.artifactId}</finalName>
```
Comment (ThomasHaas) [inline dartagnan/pom.xml:656]: What's the use of this? Why didn't we need it before?

Comment (hernanponcedeleon) [reply dartagnan/pom.xml:656]: By default the name of the generated jar is `${project.artifactId}-${version}` thus I override the file name to remove the version

**Review by ThomasHaas — APPROVED** [review]
LGTM

**Pull request merged**